docs/source/contributing/development-environment.rst

```diff
@@ -200,13 +214,20 @@ The source code and developer documentation is
 R Setup
 -------
 
+First, build a debug binary that works with R. (Note that the resulting binary will not work with Python.)
```

Comment:
It will mostly work with python, just not Polars. I guess user doesn't really need to know, though.

Reply:
I guess we could say "not suitable for Python development" to be more strictly accurate? We can revisit the wording later. Thanks for the clarification!